tutorials/colors/colorbar_only.py

```diff
@@ -0,0 +1,108 @@
+"""
+=============================
+Customized Colorbars Tutorial
+=============================
+
+This tutorial shows how to build colorbars without an attached plot.
+
+"""
+
+###############################################################################
+# Customized Colorbars
+# ====================
+#
+# ColorbarBase derives from ScalarMappable and puts a colorbar in a specified
+# axes, so it has everything needed for a standalone colorbar. It can be used
+# as is to make a colorbar for a given colormap and does not need a mappable
+# object like an image. In this tutorial we will explore what can be done with
+# standalone colorbar.
+#
+# We will start by making a figure of desired size and adding thress axes.
```

Comment:
Is `ScalarMappable` something we can link to via `:mod:`? If so we should do it, it's going to be unclear to most people what this is (if it's important)